### ISSUE TITLE
Update resources blog site from jekyll version 3 to version 4

### ISSUE THREAD

**Issue by k-doering-NOAA** [noaa-fisheries-integrated-toolbox/FIT-committee#53]

**Comment by k-doering-NOAA** [comment]
Upon investigation, it looks like the [GitHub pages plugin runs using Jekyll 3.9.3](https://pages.github.com/versions/) (more info on [how jekyll works on github pages and how to use it with other versions of jekyll](https://www.moncefbelyamani.com/making-github-pages-work-with-latest-jekyll/)). 

I'm still reading through to see if we want to update.

**Comment by k-doering-NOAA** [comment]
It doesn't seem like Jekyll 4 has anything that will drastically affect our site, so sticking with the GitHub pages plug that uses Jekyll 3.9.3 seems like the best solution.

One change that does seem helpful is using a [remote theme](https://github.com/mmistakes/minimal-mistakes#remote-theme-method), which was not set up correctly in the repository. I have now set it up on a branch and removed any boiler plate files that we haven't customized. This greatly simplifies the files in the codebase (including the gemfile, which is now much shorter).

Given that the goals of this were really 1) to update as much as possible to reduce the number of dependabot alerts; and 2) simplify the codebase in the repository, I switching to the remote theme's latest version satisfies the requirements.

I added changes to a branch for now: https://github.com/noaa-fisheries-integrated-toolbox/resources/tree/update-remote-theme

Remaining work before opening a pull request is:

- [ ] Document how remote themes work in the readme.
- [ ] Double check that site is rendering correctly by building locally
- [ ] Double check google analytics code will work
